instrumentation/github.com/Shopify/sarama/otelsarama/example/consumer/consumer.go

```diff
@@ -37,7 +37,8 @@ var (
 )
 
 func main() {
-	example.InitTracer()
+	tp := example.InitTracer()
+	defer tp.ForceFlush(context.Background())
```

Comment:
Should this `Shutdown()` instead of `ForceFlush()`?

Reply:
done